wiki/markdown/usage.md

```diff
@@ -0,0 +1,6 @@
+## Usage
+To get information about configuration and usage of REMReM Publish CLI see [here](https://eiffel-community.github.io/eiffel-remrem-publish/cliUsage.html).
+
+To get information about configuration and usage of REMReM Publish Service see [here](https://eiffel-community.github.io/eiffel-remrem-publish/serviceUsage.html).
```

Comment:
I want the usage pages also converted to markdown since they are the most important ones for user when looking in the documentation. If they're in markdown on master they will be easier to find.